index.js

```diff
@@ -9,7 +9,7 @@ if (process.env.NODE_ENV == "dev" || process.env.NODE_ENV == "development") {
     errorhandlerOptions.dumpExceptions = true;
     errorhandlerOptions.showStack = true;
 }
-
+app.enable("etag");
```

Comment:
Given the app is restarted every time there is a change in the DB, we're not going to get anything out of this unless the etags are stored somewhere. If so, where?